packages/amplify-provider-awscloudformation/src/push-resources.js

```diff
@@ -68,7 +68,7 @@ async function run(context, resourceDefinition) {
     await postPushGraphQLCodegen(context);
     await amplifyServiceManager.postPushCheck(context);
 
-    if (resources.length > 0) {
+    if (resources.concat(resourcesToBeDeleted).length > 0) {
```

Comment:
aren't resources all the resources - add/update/delete?

Reply:
no, if you look above resources is just a concatenation of add and update which was the root cause of the bug